packages/performance/src/services/transport_service.ts

```diff
@@ -25,14 +25,18 @@ const INITIAL_SEND_TIME_DELAY_MS = 5.5 * 1000;
 const DEFAULT_REMAINING_TRIES = 3;
 let remainingTries = DEFAULT_REMAINING_TRIES;
 
+interface LogResponseDetails {
+  responseAction?: string;
+}
+
 interface BatchEvent {
   message: string;
   eventTime: number;
 }
 
 /* eslint-disable camelcase */
-// CC accepted log format.
-interface CcBatchLogFormat {
+// CC/Transport accepted log format.
```

Comment:
Happy to continue discussion on naming convention.

Reply:
After discussion, I have changed the name of `Transport` to `FL` to avoid confusion. Please take a look and see whether it makes sense to you. Thank you both!